src/app/api/routes/devices.py

```diff
@@ -39,7 +39,7 @@ async def delete_device(device_id: int = Path(..., gt=0), _=Security(get_current
 
 
 @router.get("/my-devices", response_model=List[DeviceOut])
-async def fetch_my_devices(me: UserRead = Security(get_current_user, scopes=["me"])):
+async def fetch_my_devices(me: UserRead = Security(get_current_user, scopes=["admin me"])):
```

Comment:
I will check it out, but scopes should not it be a list ? 

```suggestion
async def fetch_my_devices(me: UserRead = Security(get_current_user, scopes=["admin",  "me"])):
```

Same comment for all changes

Reply:
Indeed, as one can observe in the `deps.py` file

```python

    for scope in security_scopes.scopes:
        if scope not in token_data.scopes:
            raise unauthorized_exception("Permission denied", authenticate_value)
```

Reply:
My bad, I had the string in mind of the `scopes` field in the DB